README.md

```diff
@@ -1 +1,149 @@
+To get a local copy up and running, follow these steps.
+### Prerequisites
+In order to run this project you need:
+- Visual Studio Code
+- Github Account
+- Git Bash
+- Node JS
+- Jest library
+### Setup
+Clone this repository to your desired folder: https://github.com/izebiafe/Blefi.git
+### Install
+Install this project with:
+```sh
+  cd Blefi
+```
+```sh
+  npm install 
+```
+```sh
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+### Usage
+To run the web application, execute the following command: 
+Example command:
+```sh
+  npm run dev
+```
+### Run tests
+To run tests, run the following command: npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}" --fix
+```
+```sh
+  npx eslint "**/*.js" --fix
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Presentation <a name="Presentation"></a>
+<!-- Presentation Video -->
+- This is 'Blefi presentation'<a href="https://drive.google.com/file/d/1MHhrG97FA1rZUoE5seAqxrugmNUlvloV/view">Presentation Video</a>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 👥 Authors <a name="authors"></a>
+👤 **Afimia Fidelis Izebiafe**
+- GitHub: [izebiafe]https://github.com/izebiafe
+- Twitter: [Afimia_fidelis](https://twitter.com/Afimia_fidelis)
+- LinkedIn: [Izebiafe](https://linkedin.com/in/Afimia fidelis/)
+👤 **Blessing Michael**
+- GitHub: [blessing-michael](https://github.com/blessing-michael)
+- Twitter: [toonmystory](https://twitter.com/toonmystory)
+- LinkedIn: [Blessing Asuquo](https://www.linkedin.com/in/blessing-asuquo-450991a2/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **More section inclusiveness**
+- [ ] **Desktop Responsiveness**
+- [ ] **Animation**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/izebiafe/Blefi/issues/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## ⭐️ Show your support <a name="support"></a>
+If you enjoy this project, I kindly request you to give it a star and consider following my social media pages.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+This webapp is inspired by the movies from https://github.com/izebiafe/Blefi/issues.
+We would like to express our heartfelt gratitude to Microverse for providing us with this incredible project opportunity.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) license
+d.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- You did well so far, but your readme seems muddled up on my PC, please confirm from your end that it displays properly 🙏 
![wefwef](https://github.com/Izebiafe/BleFi/assets/73607512/b64d26a8-74bd-4af9-aa18-689d6e37b97f)

Reply:
@mckent05 thank you so much for your review.
But every thing seem to work really well from my end and my README also work properly.
go over the code again and check if you might have missed something. thank you again for the review.

cc @blessing-michael

Reply:
- Okay @Izebiafe  I suggest you work on the first required change and then you can submit it for a review, another reviewer will pick it up and approve it, if required. Thank you